diff --git a/node_modules/mises-js-sdk/dist/mises-js-sdk.umd.js b/node_modules/mises-js-sdk/dist/mises-js-sdk.umd.js
index 65d2207..22412ae 100644
--- a/node_modules/mises-js-sdk/dist/mises-js-sdk.umd.js
+++ b/node_modules/mises-js-sdk/dist/mises-js-sdk.umd.js
@@ -2,7 +2,7 @@
     typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory(require('crypto'), require('buffer'), require('path'), require('fs'), require('stream'), require('events'), require('util'), require('http'), require('https'), require('url'), require('assert'), require('tty'), require('os'), require('zlib'), require('net'), require('tls')) :
     typeof define === 'function' && define.amd ? define(['crypto', 'buffer', 'path', 'fs', 'stream', 'events', 'util', 'http', 'https', 'url', 'assert', 'tty', 'os', 'zlib', 'net', 'tls'], factory) :
     (global = typeof globalThis !== 'undefined' ? globalThis : global || self, global.misesJsSdk = factory(global.require$$0$3, global.require$$0$4, global.require$$0$5, global.require$$1$1, global.require$$0$6, global.require$$0$7, global.require$$1$2, global.require$$1$4, global.require$$2$1, global.require$$7$1, global.require$$4$1, global.require$$1$3, global.require$$0$8, global.require$$8$1, global.require$$3$1, global.require$$4$2));
-})(this, (function (require$$0$3, require$$0$4, require$$0$5, require$$1$1, require$$0$6, require$$0$7, require$$1$2, require$$1$4, require$$2$1, require$$7$1, require$$4$1, require$$1$3, require$$0$8, require$$8$1, require$$3$1, require$$4$2) { 'use strict';
+})(this, (function (require$$0$3, require$$0$4, require$$0$5, require$$1$1, require$$0$6, require$$0$7, require$$1$2, require$$1$4, require$$2$1, require$$7$1, require$$4$1, require$$1$3, require$$0$8, require$$8$1, require$$3$1, require$$4$2) {
 
     function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }
 
diff --git a/node_modules/mises-js-sdk/package.json b/node_modules/mises-js-sdk/package.json
index 83274f3..6a64f05 100644
--- a/node_modules/mises-js-sdk/package.json
+++ b/node_modules/mises-js-sdk/package.json
@@ -156,5 +156,41 @@
       "pre-commit": "lint-staged",
       "pre-push": "npm run test:prod && npm run build"
     }
+  },
+  "react-native": {
+    "zlib": "browserify-zlib",
+    "crypto": "react-native-crypto",
+    "net": "react-native-tcp",
+    "http": "@tradle/react-native-http",
+    "https": "https-browserify",
+    "os": "react-native-os",
+    "fs": "react-native-level-fs",
+    "_stream_transform": "readable-stream/transform",
+    "_stream_readable": "readable-stream/readable",
+    "_stream_writable": "readable-stream/writable",
+    "_stream_duplex": "readable-stream/duplex",
+    "_stream_passthrough": "readable-stream/passthrough",
+    "stream": "stream-browserify",
+    "tty": "tty-browserify",
+    "vm": "vm-browserify",
+    "tls": false
+  },
+  "browser": {
+    "zlib": "browserify-zlib",
+    "crypto": "react-native-crypto",
+    "net": "react-native-tcp",
+    "http": "@tradle/react-native-http",
+    "https": "https-browserify",
+    "os": "react-native-os",
+    "fs": "react-native-level-fs",
+    "_stream_transform": "readable-stream/transform",
+    "_stream_readable": "readable-stream/readable",
+    "_stream_writable": "readable-stream/writable",
+    "_stream_duplex": "readable-stream/duplex",
+    "_stream_passthrough": "readable-stream/passthrough",
+    "stream": "stream-browserify",
+    "tty": "tty-browserify",
+    "vm": "vm-browserify",
+    "tls": false
   }
 }
\ No newline at end of file
