diff --git a/node_modules/pbkdf2/lib/default-encoding.js b/node_modules/pbkdf2/lib/default-encoding.js
index 3042099..dea882c 100644
--- a/node_modules/pbkdf2/lib/default-encoding.js
+++ b/node_modules/pbkdf2/lib/default-encoding.js
@@ -1,8 +1,9 @@
 var defaultEncoding
+var globalProcess = global.process
 /* istanbul ignore next */
-if (global.process && global.process.browser) {
+if (globalProcess && globalProcess.browser) {
   defaultEncoding = 'utf-8'
-} else if (global.process && global.process.version) {
+} else if (globalProcess && globalProcess.version) {
   var pVersionMajor = parseInt(process.version.split('.')[0].slice(1), 10)
 
   defaultEncoding = pVersionMajor >= 6 ? 'utf-8' : 'binary'
