diff --git a/node_modules/swappable-obj-proxy/src/createEventEmitterProxy.js b/node_modules/swappable-obj-proxy/src/createEventEmitterProxy.js
index d836e0a..523b005 100644
--- a/node_modules/swappable-obj-proxy/src/createEventEmitterProxy.js
+++ b/node_modules/swappable-obj-proxy/src/createEventEmitterProxy.js
@@ -31,6 +31,7 @@ module.exports = function createEventEmitterProxy (initialTarget, opts) {
   return proxy
 
   function setTarget(newTarget) {
+    if(!target) return
     const oldTarget = target
     target = newTarget
     // migrate listeners
